### ISSUE TITLE
`base.ParseURL()` does not exist

### ISSUE THREAD

**Issue by mateothegreat** [bluenviron/gortsplib#464]
Greetings! 👋 

Noticed something that you're probably already working on but figured I'd reach out just in case..

`base.ParseURL()` does not exist per https://github.com/bluenviron/gortsplib/blob/93b02bc0e851df641bc20a2f488bffccad5994ed/examples/client-read-format-h265-save-to-disk/main.go#L22

```
~/workspace/examples/client-read-format-h265-save-to-disk ➜ go run .  
# github.com/mateothegreat/client-read-format-h265-save-to-disk
./main.go:22:17: undefined: base.ParseURL
```

I have a repro at https://github.com/mateothegreat/client-read-format-h265-save-to-disk.

Let me know if I can help..
Thanks for all your hard work!! 
-Matthew

**Comment by kemalcanbora** [comment]
Hi Mateo,

You can use `"github.com/bluenviron/gortsplib/v4/pkg/url"` instend of `"github.com/bluenviron/gortsplib/v4/pkg/base"`

**Comment by aler9** [comment]
base.URL is a new feature that has not been included in a tag yet. The problem will solve by itself within some days.

**Comment by aler9** [comment]
fixed in v4.5.0

**Comment by mateothegreat** [comment]
Thank you!!!
